### ISSUE TITLE
Card search just doesn't work well

### ISSUE THREAD

**Issue by mattlowman** [zzorba/marvelsdb#134]
It will not list cards of selected aspects/types (i.e. aggression, justice, protection, leadership, basic, campaign, hero, encounter) and the UI for that section is vague: I can't tell if I'm meant to darken the ones I want to search in or to darken the ones that I want to exclude.

**Comment by Kamalisk** [comment]
I agree it is not super clear, but it defaults to all types selected, and you unselect what you don't want.
